eth/config.go

```diff
@@ -59,7 +59,8 @@ var DefaultConfig = Config{
 		GasPrice: big.NewInt(params.GWei),
 		Recommit: 3 * time.Second,
 	},
-	TxPool: core.DefaultTxPoolConfig,
+	TxPool:    core.DefaultTxPoolConfig,
+	RPCGasCap: 50000000,
```

Comment:
Will this cap a little bit too high? Now the gas limit of block is ~12M. Perhaps we can set it to 25M(for a single transaction).

Reply:
sgtm